mmv1/third_party/terraform/data_sources/data_source_google_container_engine_versions.go

```diff
@@ -117,13 +122,24 @@ func dataSourceGoogleContainerEngineVersionsRead(d *schema.ResourceData, meta in
 		return fmt.Errorf("Error setting default_cluster_version: %s", err)
 	}
 
-	channels := map[string]string{}
-	for _, v := range resp.Channels {
-		channels[v.Channel] = v.DefaultVersion
+	releaseChannelDefaultVersion := map[string]string{}
+	releaseChannelLatestVersion := map[string]string{}
+	for _, channelResp := range resp.Channels {
+		releaseChannelDefaultVersion[channelResp.Channel] = channelResp.DefaultVersion
+		for _, v := range channelResp.ValidVersions {
+			if strings.HasPrefix(v, d.Get("version_prefix").(string)) {
+				releaseChannelLatestVersion[channelResp.Channel] = v
+				break
+			}
+		}
 	}
-	if err := d.Set("release_channel_default_version", channels); err != nil {
+
+	if err := d.Set("release_channel_default_version", releaseChannelDefaultVersion); err != nil {
 		return fmt.Errorf("Error setting release_channel_default_version: %s", err)
 	}
+	if err := d.Set("release_channel_latest_versions", releaseChannelLatestVersion); err != nil {
```

Comment:
This is causing a panic because it doesn't match the field name.

```suggestion
	if err := d.Set("release_channel_latest_version", releaseChannelLatestVersion); err != nil {
```